presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -174,6 +178,30 @@ public void testTimestamps()
         assertQueryUsingH2Cte("SELECT COUNT(*) FROM lineitem_ex WHERE order_timestamp is not null AND year(ship_timestamp) = 1993");
     }
 
+    @Test
+    public void testFloats()
+    {
+        assertQueryUsingH2Cte("SELECT discount_real FROM lineitem_ex");
+
+        assertQueryUsingH2Cte("SELECT discount_real FROM lineitem_ex WHERE discount_real < 0.08");
+
+        assertQueryUsingH2Cte("SELECT count(*) FROM lineitem_ex WHERE tax_real > 0.1");
+
+        assertQueryUsingH2Cte("SELECT count(*) FROM lineitem_ex WHERE tax_real IS NOT NULL");
+
+        assertQueryUsingH2Cte("SELECT count(*) FROM lineitem_ex WHERE tax_real IS NULL");
+
+        assertQueryUsingH2Cte("SELECT discount_real, tax_real FROM lineitem_ex");
```

Comment:
this query is not necessary as there are `SELECT * FROM lineitem_ex` queries in other tests